models/preference.go

```diff
@@ -9,12 +9,13 @@ import (
 
 // K8SConfig represents all the k8s session config
 type K8SConfig struct {
-	InClusterConfig   bool   `json:"inClusterConfig,omitempty"`
-	K8Sfile           string `json:"k8sfile,omitempty"`
-	Config            []byte `json:"config,omitempty"`
-	Server            string `json:"configuredServer,omitempty"`
-	ContextName       string `json:"contextName,omitempty"`
-	ClusterConfigured bool   `json:"clusterConfigured,omitempty"`
+	InClusterConfig   bool         `json:"inClusterConfig,omitempty"`
+	K8Sfile           string       `json:"k8sfile,omitempty"`
+	Config            []byte       `json:"config,omitempty"`
+	Server            string       `json:"configuredServer,omitempty"`
+	ContextName       string       `json:"contextName,omitempty"`
```

Comment:
Is CurrentContext a more accurate descriptor?

Reply:
Yes it is. I would have called it the same but as this was already there so I kept the changes to the minimum. Should I change the name now?